core/mbed/source/interrupts.cpp

```diff
@@ -106,3 +106,38 @@ uint32_t vIRQ_GetPriority(uint32_t irqn)
         return UVISOR_SVC(UVISOR_SVC_ID_IRQ_PRIO_GET, "", irqn);
     }
 }
+
+/* FIXME when the number of system interrupts will be made available as
+ * a preprocessor define in a uniform way it will be deleted from here */
+#define UVISOR_NVIC_IRQn_OFFSET 16
+
+int vIRQ_GetLevel(void)
+{
+    /* if uVisor is disabled we use the standard priority levels */
+    if (__uvisor_mode == 0) {
+        /* check if an IRQn is active (an ISR is being served) */
+        uint32_t ipsr = __get_IPSR();
+        int irqn = (int) (ipsr & 0x1FF) - UVISOR_NVIC_IRQn_OFFSET;
```

Comment:
Ensure that mask is appropriate for selected target. The NVIC->IABR register (as use in NVIC_GetActive) is only 8 DWORD wide - which leaves only 256 interrupts. 0x1FF would reult in a overflow.